includes/functions.php

```diff
@@ -12,7 +12,13 @@ function URLtoPath($url, $baseURL = '', $domainBasedCaching = false)
 
         // query strings are not yet supported so we need to bail is there is one present
         if (!empty($urlParts['query'])) {
-            return;
+            parse_str($urlParts['query'], $queryParts);
+            if (!empty($queryParts['stage']) && $queryParts['stage'] === 'Live') {
+                unset($queryParts['stage']);
+            }
+            if (!empty($queryParts)) {
```

Comment:
what's this check for?

Reply:
Reverting, this fix is not needed anymore.